app/javascript/components/student/tracks-list/StatusFilter.jsx

```diff
@@ -1,22 +1,26 @@
 import React from 'react'
 
-export function StatusFilter({ options, dispatch }) {
-  function handleChange(e) {
-    dispatch({ type: 'status.changed', payload: { status: e.target.value } })
+export function StatusFilter({ options, dispatch, ...props }) {
```

Comment:
To me the `props` argument implies that we don't necessarily care about the props' individual values but just about all the props. What if we would do:

```suggestion
export function StatusFilter({ options, dispatch, value }) {
  const currentValue = value || options[0].value
```

This still passes the tests (well, if we use `{option.value === currentValue}` later on).